src/inverseRobot.js

```diff
@@ -25,6 +25,20 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const keys = Object.values(robot);
+  const values = Object.keys(robot);
+  const newRobot = {};
+  const len = keys.length;
+
+  for (let i = 0; i < len; i++) {
+    if (keys.slice(i + 1, keys.length).includes(keys[i])) {
```

Comment:
Same here, that's an overkill, use Object methods to check if object has such property, [Checklist](https://github.com/mate-academy/js_inverse_robot/blob/master/checklist.md) 2nd item